test/jdk/javax/net/ssl/ServerName/EndingDotHostname.java

```diff
@@ -0,0 +1,248 @@
+/*
+ * Copyright (C) 2022 THL A29 Limited, a Tencent company. All rights reserved.
+ * DO NOT ALTER OR REMOVE NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2 as
+ * published by the Free Software Foundation. THL A29 Limited designates
+ * this particular file as subject to the "Classpath" exception as provided
```

Comment:
Please remove the Classpath part. It does not apply to tests. Same comment to the other file.

Reply:
Ooops, I missed it.  Thank you!  Updated.